deployments/nasa/config/common.yaml

```diff
@@ -1,5 +1,12 @@
 pangeo:
   jupyterhub:
+    scheduling:
+      userPods:
+        nodeAffinity:
+          matchNodePurpose: require
+      corePods:
+        nodeAffinity:
+          matchNodePurpose: require
```

Comment:
Nice! What labels / taints do you have on your nodes?

Reply:
```
    labels:
      node-role.kubernetes.io/role: hub
      hub.jupyter.org/node-purpose: core

    labels:
      node-role.kubernetes.io/role: user-notebook
      hub.jupyter.org/node-purpose: user
    taints:
      hub.jupyter.org/dedicated: 'user:NoSchedule'

    labels:
      node-role.kubernetes.io/role: dask-worker
      k8s.dask.org/node-purpose: worker
    taints:
      k8s.dask.org/dedicated: 'worker:NoSchedule'

```

Reply:
whoohoo!